LICENSE.md

```diff
@@ -4021,6 +4021,12 @@ License: [The Apache Software License, Version 2.0](http://www.apache.org/licens
 
 ===========================================================================
 
+Mapbox Navigation uses portions of the Activity Kotlin Extensions (Kotlin extensions for 'activity' artifact).
```

Comment:
All of the added sections are duplicated now, why are these changes necessary?

Reply:
Honestly was letting the license script do the work, it was failing https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-android/20930/workflows/078c27e8-d0d5-4775-9e9b-0e5147b411ab/jobs/106520

Reply:
We should investigate, I see that kotlinx-coroutines-core is included 17 times here!

Reply:
OK figured out the cause https://github.com/mapbox/mapbox-navigation-android/pull/6336/files#diff-4673a3aba01813b595de187a7a6e9e63a3491d55821606fecd9f13a10c188a1dL4066-L4083

it's in the difference between `api(project(":libnavigation-android"))` and `api("com.mapbox.navigation:android:${carNavVersion}")`

When including an `api(project` the licenses are merged. When including a library with the specific version, all the dependencies are duplicated.

Considering androidauto is a libarary in isolation, considering the idea that we generate a separate license.

But for now I think this duplication is fine.

Reply:
Looks like it affects all of the modules in the project, not just Android Auto. Which explains why kotlinx-coroutines-core is included 17 times.